CineasteTests/WantToSeeMoviesViewControllerTests.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  WantToSeeMoviesViewControllerTests.swift
+//  CineasteTests
+//
+//  Created by Felizia Bernutz on 27.12.17.
+//  Copyright © 2017 notimeforthat.org. All rights reserved.
+//
+
+import XCTest
+@testable import Cineaste
+
+class WantToSeeMoviesViewControllerTests: XCTestCase {
+    let wantToSeeMoviesVC = WantToSeeMoviesViewController.instantiate()
+
+    func testViewDidLoad() {
```

Comment:
I always try to name a test like the function and adding `test` in front of it. This is a test for the `viewDidLoad` which is not complete yet, because there are some untested things in `viewDidLoad` left.